index.js

```diff
@@ -0,0 +1,26 @@
+const {
```

Comment:
instead of require each function u can require the file and use the functions like this:
const dbConn = require("...");
dbConn.checkConnectionToDb();